src/admin-views/editor/panel/ticket.php

```diff
@@ -152,7 +179,135 @@ class="tribe_soft_note ticket_form_right"
 						); ?>
 					</span>
 				</div>
-				<fieldset id="tribe_ticket_provider_wrapper" class="input_block" aria-hidden="true" >
+				<div class="input_block">
+					<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+					<div class="ticket_form_right">
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date ticket_field"
+							name="ticket_start_date"
+							id="ticket_start_date"
+							value="<?php echo esc_attr( $ticket ? $start_date : null ); ?>"
+							data-validation-type="datepicker"
+							data-validation-is-less-or-equal-to="#ticket_end_date"
+							data-validation-error="<?php echo esc_attr( json_encode( $start_date_errors ) ) ?>"
+						/>
+						<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'event-tickets' ) ?></span>
```

Comment:
:no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 831ba76cac37553dc2d8c4587c2120b1</sub>

Reply:
Same here.

Reply:
done in both places

Reply:
```suggestion
						<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'event-tickets' ); ?></span>
```